### ISSUE TITLE
Does this library work on RN Android?

### ISSUE THREAD

**Issue by rtkhanas** [davidkpiano/react-redux-form#701]
It is mentioned in docs that it works with all RN iOS form controls, but what about Android? 

Thank you!

**Comment by davidkpiano** [comment]
I don't think so yet, but it would be very simple to copy [`src/native.js`](https://github.com/davidkpiano/react-redux-form/blob/master/src/native.js) and adapt it for Android controls. Want to do a PR? 😄

**Comment by rtkhanas** [comment]
Yes, I will try to check this as soon as I have a bit of free time!

**Comment by januszdotnl** [comment]
Any luck @rtkhanas ?

In the mean time, I'll give it a shot @davidkpiano ... Once you go RRF, you never go back :smile:

**Comment by rtkhanas** [comment]
I'm not using this project yet, so had to change to look at this issue. 😢

**Comment by davidkpiano** [comment]
Thanks @januszdotnl ! It should be pretty straightforward - the implementation is very similar to React Native (iOS).

**Comment by TheMightyPenguin** [comment]
@januszdotnl hey have you done any work on this? I'm thinking on giving it a try to ope a PR! 😄

**Comment by Jahans3** [comment]
@TheMightyPenguin did you open a PR in the end? :)

**Comment by davidkpiano** [comment]
@Jahans3 feel free to tackle this if you know how it'd be done 😉

**Comment by Jahans3** [comment]
@davidkpiano submitted a PR :-)
